src/Sylius/Bundle/ApiBundle/spec/Security/OrderAdjustmentsVoterSpec.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace spec\Sylius\Bundle\ApiBundle\Security;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\AdjustmentInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Sylius\Component\Core\Model\ShopUser;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\VoterInterface;
+
+final class OrderAdjustmentsVoterSpec extends ObjectBehavior
+{
+    function it_does_not_support_other_attributes(): void
+    {
+        $this->supportsAttribute('OTHER_ATTRIBUTE')->shouldReturn(false);
+    }
```

Comment:
```suggestion
    function it_only_supports_sylius_order_adjustment_attribute(): void
    {
        $this->supportsAttribute('SYLIUS_ORDER_ADJUSTMENT')->shouldReturn(true);
        $this->supportsAttribute('OTHER_ATTRIBUTE')->shouldReturn(false);
    }
```
Also, use the `const` throughout this spec